### PR TITLE
Fix remaining end-to-end tests for docker-compose refactor

### PR DESCRIPTION
**Pull request by hugoesthere**
✅ 

Should close out: https://github.com/Originate/exosphere-sdk/issues/272

### CODE REVIEW

File: cli/features/tutorial.feature [hunk 1]
```diff
@@ -266,7 +266,9 @@ Feature: Following the tutorial
       """
     When running "exo setup" in this application's directory
     And starting "exo run" in this application's directory
-    And waiting until I see "all services online" in the terminal
+    # TODO: change back to 'And waiting until I see "all services online" in the terminal'
+    # when online text monitoring is implemented
+    And waiting until I see "HTML server is running" in the terminal
```
Comment (hugoesthere) [inline cli/features/tutorial.feature:271]: I thought this was a better solution than waiting a set amount of time. Open to alternative suggestions.

File: cli/features/tutorial.feature [hunk 2]
```diff
@@ -266,7 +266,9 @@ Feature: Following the tutorial
       """
     When running "exo setup" in this application's directory
     And starting "exo run" in this application's directory
-    And waiting until I see "all services online" in the terminal
+    # TODO: change back to 'And waiting until I see "all services online" in the terminal'
```
Comment (ben-goldberg) [inline cli/features/tutorial.feature:269]: should we track this with an issue so the TODO doesn't get lost?

**Review by ben-goldberg — APPROVED** [review]

File: cli/features/tutorial.feature (hunk 2 above)
Comment (hugoesthere) [reply cli/features/tutorial.feature:269]: Yep, just made one: https://github.com/Originate/exosphere-sdk/issues/286

**Review by trushton — APPROVED** [review]
Nicely done with all of this docker-compose work

**Pull request closed without merging**